dwbb/src/interface/DW_ram_2r_w_a_dff.scala

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2024 Jiuyang Liu <liu@jiuyang.me>
+package org.chipsalliance.dwbb.interface.DW_ram_2r_w_a_dff
+
+import chisel3._
+import chisel3.experimental.SerializableModuleParameter
+import chisel3.util.log2Ceil
+import upickle.default
+
+object Parameter {
+  implicit def rw: default.ReadWriter[Parameter] =
+    upickle.default.macroRW[Parameter]
+}
+
+case class Parameter(
+    dataWidth: Int = 4,
+    depth: Int = 8,
+    rstMode: Boolean = true
```

Comment:
what is rstMode

Reply:
It would be clearer and simpler to be a `Boolean` type.